gui/src/installer/view/editor/mod.rs

```diff
@@ -128,6 +128,39 @@ pub fn path(
     .into()
 }
 
+pub fn uneditable_defined_key<'a>(
```

Comment:
This and the `defined_key` function seem quite similar so would it be worth creating a common function with an `editable` parameter, say?

Reply:
I was thinking about it, but it felt not good to have multiple bool args (with the `fixed` arg) that concern multiple templates for the single safe multisig case.